### ISSUE TITLE
What happen to the icons?

### ISSUE THREAD

**Issue by smplyjr** [equinusocio/vsc-material-theme#333]
Hi @equinusocio, this one is probably not a bug nor an issue. We can consider this as a question and I did post it here instead of posting to somewhere else, to get an answer directly from the creator of course.

As far as I know, visual studio code team agreed that the icons will not be included on each every theme available in the market place. I downloaded the AppIcon you provided but no luck on how to install it.

I like the icon you made. I hope I can still use it.

**Comment by equinusocio** [comment]
Hi @smplyjr. Icons are now shipped as separated extension. Search for "Material Theme Icons"
https://marketplace.visualstudio.com/items?itemName=Equinusocio.vsc-material-theme-icons

**Comment by smplyjr** [comment]
Thanks @equinusocio, really appreciated your good work.

**Comment by equinusocio** [comment]
I read again your issue, do you mean the vscode app icon or the file icons?

**Comment by smplyjr** [comment]
I mean, since the icons are missing by default (when I installed the theme). I hurried up heading to this repository and checked the readme file if what's wrong with the icons. Then I found you have

**Other resources:**
- AppIcon: [Download](https://github.com/equinusocio/vsc-material-theme/files/989048/vsc-material-theme-appicon.zip) the official Material Theme app icon for Visual Studio code

So, I did download that AppIcon assuming that this is the icons that I am looking for. But unfortunately, I do not know how to install it. So, I post a question here.

I'm not sure what's the difference of app icons from file icons. The image attached is the one I'm looking.

![Icons](https://user-images.githubusercontent.com/16571669/54648170-28412b00-4ae0-11e9-8334-81494c75799b.PNG)

I think it will better mate if you update the readme file of this repo and let us know that there is **Material Theme Icons** available in the market.
